src/Components/Alert/AlertProvider.tsx

```diff
@@ -115,26 +118,33 @@ export const AlertProvider: FC<AlertProviderProps> = ({
 
   const handleComplete = async () => {
     try {
-      const reponse = await submitMutation({
+      const reponse = await submitCreateAlert({
         variables: {
           input: {
-            attributes: state.criteria,
-            userAlertSettings: state.settings,
-          },
+            ...state.criteria,
+            settings: state.settings,
+          } as { artistIDs: string[] }, // artistIDs is required in the input type
```

Comment:
@anandaroop I wound up backing out of that change that makes `artistIDs` required and not null/undefined. Essentially, there's some liberal sharing of types, broadly, between artwork filter _and_ alert criteria code, as well as objects being typed as such being used in various contexts (such as a 'create alert' mutation).

The mutation however now requires `artistIDs` to be present (this was optional in the stitched schema), and making that property required in the more global shared type led to lots of strange fall-out in unexpected places, so just went with a better typing closer to where it's needed here.